wm.c

```diff
@@ -3,6 +3,7 @@
 
 #include "config.h"
 
+#include <X11/X.h>
```

Comment:
why is this needed?

Reply:
welp my bad
must hae been my ide auto adding libraries
I'll clean it up

Reply:
Welp my bad
Must have been vscode adding libraries on it's own